nautobot/docs/release-notes/version-1.2.md

```diff
@@ -233,6 +250,7 @@ Just as with the UI, the `slug` can still always be explicitly set if desired.
 ### Fixed
 
 - [#453](https://github.com/nautobot/nautobot/issues/453) - Fixed potential `ValueError` when rendering `JobResult` detail view with non-standard `JobResult.data` contents.
+- [#616](https://github.com/nautobot/nautobot/issues/616) - A CustomField with a type of "text" will only validate if the provided value is a string.
```

Comment:
```suggestion
```

Reply:
I'm confused by this? I see #616 as closed.

Reply:
Nevermind. I see it in there twice.